Utilities/QC/QCMetricsExtractor/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# @author  Barthélémy von Haller
+
+set(MODULE_NAME "QCMetricsExtractor")
+
+O2_SETUP(NAME ${MODULE_NAME})
+
+# Define the source and header files
+set(SRCS
+    src/MetricsExtractor.cxx
+    )
+
+set(HEADERS
+    include/${MODULE_NAME}/MetricsExtractor.h
+    )
+
+set(LIBRARY_NAME ${MODULE_NAME})
+set(BUCKET_NAME QC_base_bucket)
+
+O2_GENERATE_LIBRARY()
+
+# todo we repeat ourselves because the above macro dares deleting the variables we pass to it.
+set(LIBRARY_NAME ${MODULE_NAME})
+set(BUCKET_NAME QC_merger_bucket)
```

Comment:
@PatrykLesiak this is probably a mistake, in line 17 the QC_base_bucket is used. It's anyhow strange to set the BUCKET_NAME twice, but it needs to be consistent.

I hope we can fix this error prone double-set in the framework macros